src/Microsoft.ComponentDetection.Detectors/vcpkg/Contracts/Annotation.cs

```diff
@@ -0,0 +1,13 @@
+namespace Microsoft.ComponentDetection.Detectors.Vcpkg.Contracts
+{
+    public class Annotation
+    {
+        public string Date { get; set; }
```

Comment:
Why are we having a date as String?

Reply:
Entirely because I'm not familiar enough with C# to know all the possible Date edge cases and don't have any plans to use the information. The format is specified to be ISO 8601 (e.g. `2010-01-29T18:30:22Z`) -- could you suggest code to correctly handle that?

Reply:
Sure I can push a change for that in a bit...